src/main/java/com/regalii/regaliator/RegaliatorXdataV32.java

```diff
@@ -0,0 +1,127 @@
+package com.regalii.regaliator;
```

Comment:
not sure if this should be part of the source, as it is only an example, if they include and package this, they will be always included

Reply:
Right... I just thought it might make things easier to start testing right away. @GRoguelon what are your thoughts on this?